README.md

```diff
@@ -190,7 +191,8 @@ For more information on Android development, see the Android developer site at:
 
 ## Talk to Us
 
-Visit our GitHub [Issues](https://github.com/aws-amplify/aws-sdk-android/issues) to leave feedback and to connect with other users of the SDK.
+* Visit our GitHub [Issues](https://github.com/aws-amplify/aws-sdk-android/issues) to leave feedback and to connect with other users of the SDK.
+* Join our [Discord Community](https://discord.gg/jWVbPfC)
```

Comment:
In the above we're linking "Issues" and not "GitHub", but here we're linking "Discord" and "Community." I prefer the latter, but please make these consistent.